### PR TITLE
feat(api-client, webapp-events): add federation delete & connection remove events (WPB-203)

### PR DESCRIPTION
**Pull request by thisisamir98**
## Pull Request Checklist

- [ ] My code is covered by tests
- [ ] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

### CODE REVIEW

File: packages/api-client/src/event/FederationEvent.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import {FederationDeleteData, FederationConnectionRemovedData} from '../federation/data';
+
+export enum FEDERATION_EVENT {
+  FEDERATION_DELETE = 'federation.delete',
+  FEDERATION_CONNECTION_REMOVED = 'federation.connectionRemoved',
+}
+
+// {"type":"federation.delete", "domain": "backendB.com"}
+// {"type":"federation.connectionRemoved", "domains":["backendA.com", "backendB.com"]}
```
Comment (tlebon) [inline packages/api-client/src/event/FederationEvent.ts]: remove the comments i think
```suggestion

```

**Review by tlebon — APPROVED** [review]
seems fine

File: packages/api-client/src/event/FederationEvent.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  FEDERATION_CONNECTION_REMOVED = 'federation.connectionRemoved',
+}
+
+export type FederationEventData = FederationDeleteData | FederationConnectionRemovedData | null;
+
+export type FederationEvent = FederationDeleteEvent | FederationConnectionRemovedEvent;
+
+export interface BaseFederationEvent {
+  data: FederationEventData;
+  Federation: string;
```
Comment (V-Gira) [inline packages/api-client/src/event/FederationEvent.ts:33]: What's this key about?

Comment (thisisamir98) [reply packages/api-client/src/event/FederationEvent.ts]: nice find. deleted it. https://github.com/wireapp/wire-web-packages/pull/5340/commits/1a6721bb39dea6c912e28c6117c63043ea79ae0c

**Review by V-Gira — APPROVED** [review]

**Pull request merged**